MAUI/RadialGauge/shape-pointer.md

```diff
@@ -0,0 +1,244 @@
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfRadialGauge sfRadialGauge = new SfRadialGauge();
+
+		RadialAxis radialAxis = new RadialAxis();
+		sfRadialGauge.Axes.Add(radialAxis);
+
+		ShapePointer pointer = new ShapePointer();
+		pointer.Value = 60;
+		pointer.ShapeType = ShapeType.Circle;
+		pointer.HasShadow = true;
+		radialAxis.Pointers.Add(pointer);
+
+		this.Content = sfRadialGauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change shape pointer elevation](images/marker-pointer/pointer_shadow.PNG)
+
+## Shape pointer overlay
+
+The shape pointer overlay rendered around the shape when the shape pointer is dragged. When [`IsInteractive`]() property of pointer is set as true and while dragging the marker, the overlay will come around the marker pointer.
+
+{% tabs %}
+
+{% highlight xaml %}
+
+  <gauge:SfRadialGauge>
+                <gauge:SfRadialGauge.Axes>
+                    <gauge:RadialAxis>
+                        <gauge:RadialAxis.Pointers>
+                            <gauge:ShapePointer Value="60" ShapeType="Circle" 
+                                                IsInteractive="True" OverlayRadius="20"/>
+                        </gauge:RadialAxis.Pointers>
+                    </gauge:RadialAxis>
+                </gauge:SfRadialGauge.Axes>
+            </gauge:SfRadialGauge>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+SfRadialGauge sfRadialGauge = new SfRadialGauge();
+
+		RadialAxis radialAxis = new RadialAxis();
+		sfRadialGauge.Axes.Add(radialAxis);
+
+		ShapePointer pointer = new ShapePointer();
+		pointer.Value = 60;
+		pointer.ShapeType = ShapeType.Circle;
+		pointer.OverlayRadius = 20;
+		pointer.IsInteractive = true;
+		radialAxis.Pointers.Add(pointer);
+
+		this.Content = sfRadialGauge;
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Change shape pointer overlay](images/marker-pointer/pointer_overlay.PNG)
+
+## Marker position customization
+
+The marker pointer can be moved near or far from its actual position using the [`Offset`]() and [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) properties. 
+
+When you set [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) to pixel, the marker pointer will be moved based on the pixel value. If you set [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) to factor, then provided factor will be multiplied with the axis radius value, and then the pointer will be moved to corresponding value. The default value of [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) is [`SizeUnit.Pixel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SizeUnit.html#Syncfusion_Maui_Gauges_SizeUnit_Pixel).
```

Comment:
When you set [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) to pixel, the marker pointer is moved based on the pixel value. If you set [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) to factor, then provided factor will be multiplied by the axis radius value, and then the pointer will be moved to the corresponding value. The default value of [`OffsetUnit`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.MarkerPointer.html#Syncfusion_Maui_Gauges_MarkerPointer_OffsetUnit) is [`SizeUnit.Pixel`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Gauges.SizeUnit.html#Syncfusion_Maui_Gauges_SizeUnit_Pixel).